src/js/apps/forms/form/form-patient_app.js

```diff
@@ -45,9 +45,12 @@ export default App.extend({
     this.form = form;
     this.isReadOnly = this.form.isReadOnly();
 
+    const formWidgets = this.form.getWidgets();
```

Comment:
Actually I think ideally this is just `const widget = this.form.getWidget();` where the entity request is in the form model

Reply:
Yes it definitely can. Would be cleaner that way.